vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -211,14 +211,22 @@ function Get-PrivatePackagesToCacheOnVHD {
         $dir = "c:\akse-cache\private-packages"
         New-Item -ItemType Directory $dir -Force | Out-Null
 
+        Write-Log "Downloading azcopy"
+        Invoke-WebRequest -UseBasicParsing "https://aka.ms/downloadazcopy-v10-windows" -OutFile azcopy.zip
```

Comment:
Do you need to remove the azcopy.zip?

Reply:
Thanks for the comment. Added the remove step.